airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/typing_deduping/RedshiftSqlGeneratorIntegrationTest.java

```diff
@@ -215,4 +222,14 @@ public void testCreateTableIncremental() throws Exception {
     // TODO assert on table clustering, etc.
   }
 
+  private static String escapeStringLiteral(final String str) {
+    if (str == null) {
+      return null;
+    } else {
+      // jooq handles most things
+      // but we need to manually escape backslashes for some reason
+      return str.replace("\\", "\\\\");
```

Comment:
Their definition of backslack escaping is  quite different 😹  https://www.jooq.org/doc/latest/manual/sql-building/dsl-context/custom-settings/settings-backslash-escaping/